TECHMANIA/Assets/Scripts/Serializable/Options.cs

```diff
@@ -246,6 +247,44 @@ public static void RestoreBackup()
     {
         instance = backupInstance;
     }
+
+    public static IEnumerator ResetCustomDataLocation (bool reloadSkin = true)
+    {
+        instance.customDataLocation = false;
+        instance.noteSkin = "Default";
+        instance.vfxSkin = "Default";
+        instance.comboSkin = "Default";
+        instance.gameUiSkin = "Default";
+        instance.tracksFolderLocation = "";
+        instance.skinsFolderLocation = "";
+        instance.SaveToFile(Paths.GetOptionsFilePath());
+
+        if (reloadSkin)
+        {
+            GlobalResourceLoader resourceLoader = UnityEngine.Object.FindObjectOfType<GlobalResourceLoader>();
```

Comment:
There are a few problems with the loading of skins here.

* This sounds like something `GlobalResourceLoader` should do, not `Options`.
* If `reloadSkin` is false, then this function doesn't need to be a coroutine, yet callers still have to call `StartCoroutine`.

My suggestions is:
* Remove the skin loading code from this method.
* Make `ResetCustomDataLocation` not a coroutine.
* Modify `GlobalResourceLoader.LoadResourcesOnStartUp` a bit to be reusable. Then, if a caller needs to reload skins, call that after `ResetCustomDataLocation`.